### PR TITLE
Added FrameworkResources

### PR DESCRIPTION
**Pull request by Shqrdx**
Changes :
 -New class FrameworkResources( its internal class from XNA which contains constant strings and other resources for use in multiply places in code).
 -Added FrameworkResources.ResourceCreationWhenDeviceIsNull to all places when graphics resource is created and when graphicsDevice is null in exception message.

### CODE REVIEW

File: MonoGame.Framework/FrameworkResources.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace Microsoft.Xna.Framework
+{
+    internal static class FrameworkResources
+    {
+        #region Error strings
+
+        internal const string ResourceCreationWhenDeviceIsNull = "The GraphicsDevice must not be null when creating new resources.";
```
Comment (tomspilman) [inline MonoGame.Framework/FrameworkResources.cs]: @KonajuGames - What do you think about starting to follow this technique for message strings?

Comment (KonajuGames) [inline MonoGame.Framework/FrameworkResources.cs]: I've thought about it before, and is something we should do.  Thanks @Shqrdx for getting it started.

Comment (tomspilman) [inline MonoGame.Framework/FrameworkResources.cs]: I guess the main benefit is removing the slight differences in related error messages and being able to localize it eventually if we wanted to.

Comment (KonajuGames) [inline MonoGame.Framework/FrameworkResources.cs]: Those are the two primary reasons.

**Comment by Shqrdx** [comment]
Rebase completed. Ready for merging... I hope )

**Comment by tomspilman** [comment]
Ok... merging!

**Pull request merged**